### PR TITLE
Add ReplaceLinearOrder for USDT perp

### PR DESCRIPTION
**Pull request by quantoor**

### CODE REVIEW

File: future_usdt_perpetual.go [hunk 1]
```diff
@@ -495,6 +496,47 @@ func (s *FutureUSDTPerpetualService) LinearCancelAllOrder(param LinearCancelAllP
 	return &res, nil
 }
 
+// ReplaceLinearOrderResponse :
+type ReplaceLinearOrderResponse struct {
+	CommonResponse `json:",inline"`
+	Result         CreateLinearOrderResult `json:"result"`
```
Comment (hirokisan) [inline future_usdt_perpetual.go:502]:
```suggestion
	Result         ReplaceLinearOrderResult  `json:"result"`
```

**Comment by hirokisan** [comment]
📝 document

https://bybit-exchange.github.io/docs/futuresV2/linear/#t-replaceactive

File: future_usdt_perpetual.go [hunk 2]
```diff
@@ -495,6 +496,47 @@ func (s *FutureUSDTPerpetualService) LinearCancelAllOrder(param LinearCancelAllP
...
+	SlTriggerBy *TriggerByFuture `json:"sl_trigger_by,omitempty"`
+}
+
+// ReplaceLinearOrder :
+func (s *FutureUSDTPerpetualService) ReplaceLinearOrder(param ReplaceLinearOrderParam) (*ReplaceLinearOrderResponse, error) {
+	var res ReplaceLinearOrderResponse
+
+	body, err := json.Marshal(param)
+	if err != nil {
+		return nil, fmt.Errorf("json marshal for CreateLinearOrderParam: %w", err)
```
Comment (hirokisan) [inline future_usdt_perpetual.go:530]:
```suggestion
		return nil, fmt.Errorf("json marshal for ReplaceLinearOrderParam: %w", err)
```

**Review by hirokisan — CHANGES_REQUESTED** [review]
Thank you for your contribution!

As I commented, some changes may need to be made.

**Comment by quantoor** [comment]
My pleasure to contribute!

I have amended the commit with the requested changes.

**Review by hirokisan — APPROVED** [review]
Looks good!

Thank you for your contribution again.

**Pull request merged**